Rename Valid language tag to Known primary language tag (#1738)
act-rules/act-rules.github.io: _rules/element-lang-valid-de46e4.md

#### Failed Example 8

> The `lang` [attribute value][] of this `p` element is an [iso 639.2][] three letters code, which is not a [known primary language tag][].

Reviewer: ```suggestion
The `lang` [attribute value][] of this `p` element is an [iso 639.2][] three letters code, which has no [known primary language tag][].
```

#### Failed Example 9

> The `lang` [attribute value][] of this `p` element is a [grandfathered tag][grandfathered tags], which is not a [known primary language tag][].

Reviewer: ```suggestion
The `lang` [attribute value][] of this `p` element is a [grandfathered tag][grandfathered tags], which has no [known primary language tag][].
```
